client/src/components/dashboard/deploy/DeployPage.js

```diff
@@ -37,7 +66,7 @@ const DeployPage = ({ teamId }) => {
   };
 
   const GitHubLoginButton = (
-    <GitHubLogin
+    <GHButton
       className="float-right github-login-button"
```

Comment:
do we still need the `github-login-buton` here? cause if I remember correctly, that's the styling of the previous button (ie. change the colour to blue etc.)
If we don't need this maybe we can also remove the CSS part of `github-login-button`